### DISCUSSION TITLE
error building stella_vslam on ros

### DISCUSSION THREAD

**Discussion by mgrallos** [OpenVSLAM-Community/openvslam_ros#112] [Q&A] [answered]
Hi I am trying to implement stella_vslam but I got this error after building it on ros using catkin_make. I am using ROS noetic and it did build up until a 100 but still has an error. 

![image](https://user-images.githubusercontent.com/64775540/221413294-39b5b2f8-fccd-4381-9588-d76fa5748931.png)

**Comment by ymd-stella** [comment, marked as answer]
Use OpenCV installed by ROS without building OpenCV.

**Reply by mgrallos** [reply]
sorry how to do that? so i just have to install OpenCV and not build it?

**Reply by mgrallos** [reply]
I already added this line find_package(OpenCV REQUIRED) on my CMakeList.txt but got the same error

**Reply by ymd-stella** [reply]
> I already added this line find_package(OpenCV REQUIRED) on my CMakeList.txt but got the same error

This is because you installed a different version of OpenCV in your environment where OpenCV was installed by ROS.

Restart from reinstalling the OS.

All necessary steps for installation can be found at https://github.com/stella-cv/stella_vslam_ros/blob/ros/Dockerfile.
